lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -114,13 +121,37 @@ sub get_modified_devices_warning {
     return $self->{ModifiedDevicesWarning};
 }
 
+
+sub add_new_partition {
+    my ($self, $args) = @_;
+    if ($args->{size} eq "max") {
+        $self->get_resize_page()->press_next();
+    }
+    else {
+        $self->get_resize_page()->set_custom_size($args->{size});
```

Comment:
Ok problem is this line. TBH, I think that `add_new_partition` will work as it is in v4 Expert Partitioner. Could you please try to remove this implementation and trigger VR?

Reply:
trying this here http://waaa-amazing.suse.cz/tests/13762#live

Reply:
yep, it worked, so let's remove this change and the change to `lib/Installation/Partitioner/LibstorageNG/v4_3/ResizePage.pm`. And we are done here and will take of migrating those methods to REST API in a separate ticket.
I did same thing myself: https://openqa.suse.de/tests/5182371